src/app/page.tsx

```diff
@@ -16,28 +17,34 @@ export default function Home() {
     });
   }, []);
 
-  const onChange = (e) => {
-    const searchTerm = e.target.value;
+  const onSearchChange = (e) => {
+    const value = e.target.value;
 
-    document.getElementById("search-term").innerHTML = searchTerm;
+    setSearchTerm(value);
+
+    const newSearchTerm = value.toUpperCase();
 
-    console.log("filtering advocates...");
     const filteredAdvocates = advocates.filter((advocate) => {
```

Comment:
This realistically would be done a lot more efficiently, like on the DB side, but at least made the searches case-insensitive for now